docs/sources/datasources/loki.md

```diff
@@ -154,6 +154,9 @@ provides the following functions you can use in the `Query` input field.
 | `label_values(label)`                      | Returns a list of label values for the `label`.                                      |
 | `label_values(log stream selector, label)` | Returns a list of label values for the `label` in the specified `log stream selector`.|
 
+### Ad hoc filters variable
+Loki supports the special ad hoc filters variable type. It allows you to specify any number of label/value filters on the fly. These filters are automatically applied to all your Loki queries.
```

Comment:
is this specific to Loki only, or other data sources can use this variable? 

As a best practice, better to not use "special" in product docs.

Reply:
`Ad hoc filter variables only work with Prometheus, Loki, InfluxDB and Elasticsearch data sources.` I used the same copy as it is in Prometheus https://grafana.com/docs/grafana/latest/datasources/prometheus/#ad-hoc-filters-variable. We can remove `special` from both places.